py3dtilers/Common/README.md

```diff
@@ -0,0 +1,144 @@
+for object in objects_to_tile:
+    print(object.get_id())
+```
+
+### 🟢ObjectsToTileWithGeometry
+An [🟢](#objectstotilewithgeometry)&nbsp;_ObjectsToTileWithGeometry_ contains objects to tile ([🟣](#objectstotile)&nbsp;_ObjectsToTile_) and can have its own geometry ([🟣](#objectstotile)&nbsp;_ObjectsToTile_).
+It can be created with:
+```
+objects_to_tile_with_geom = ObjectsToTileWithGeometry(objects_to_tile, geometry) # Instance with its own geometry
+# or
+objects_to_tile_with_geom = ObjectsToTileWithGeometry(objects_to_tile) # Instance without its own geometry
+```
+
+## [kd_tree](Common/kd_tree.py)
+The kd_tree distributes the [🟠](#objecttotile)&nbsp;_ObjectToTile(s)_ contained in an [🟣](#objectstotile)&nbsp;_ObjectsToTile_ into multiple [🟣](#objectstotile)&nbsp;_ObjectsToTile_. Each instance of [🟣](#objectstotile)&nbsp;_ObjectsToTile_ can have a maximum of `maxNumObjects`:
+```
+# Takes : an ObjectsToTile
+# Returns : a list of ObjectsToTile
+distributed_objects = kd_tree(objects_to_tile, 100) # Max 100 objects per ObjectsToTile
+```
+
+## [lod_tree](https://github.com/VCityTeam/py3dtilers/blob/CityTiler_with_LodTree/py3dtilers/Common/lod_tree.py)
+lod_tree creates a tileset with a parent-child hierarchy. Each node of the tree contains an [🟣](#objectstotile)&nbsp;_ObjectsToTile_ (the geometries of the node) and a list of child nodes.
+A node will correspond to a tile (.b3dm file) of the tileset.  
+The leafs of the tree contain the geometries with the most details. The parent node of each node contains a lower level of details.
+
+The lod_tree creation takes an [🟣](#objectstotile)&nbsp;_ObjectsToTile_ (containing [🟠](#objecttotile)&nbsp;_ObjectToTile(s)_ with detailled geometries and bounding boxes) and returns a tileset.
+
+The first step of the tree creation is the distribution of [🟠](#objecttotile)&nbsp;_ObjectToTile(s)_ into groups. A group is an instance of [🟢](#objectstotilewithgeometry)&nbsp;_ObjectsToTileWithGeometry_ where the objects to tile ([🟣](#objectstotile)&nbsp;(_ObjectsToTile_) are a group of detailled geometries. The group can also have its own geometry ([🟣](#objectstotile)&nbsp;_ObjectsToTile_), which is a lower level of details of the detailled geometries.
+The groups are either created with `create_loa` or from the list of [🟣](#objectstotile)&nbsp;_ObjectsToTile_ of `kd_tree`. The groups from `create_loa` have their own geometry, those from `kd_tree` don't.
+
+To create a tileset with LOA\*, use:
+```
+create_tileset(objects_to_tile, # Objects to transform into 3Dtiles
+               also_create_loa=True, # Indicate to create a LOA
+               loa_path="./path/to/dir") # Path to a directory if additional files are needed to create LOA
+```
+\* _Level Of Abstraction_, in this case it consists in a tile with a low level of details where the geometries are grouped into one.
+
+Resulting tilesets:
+Groups from `kd_tree` which __don't have__ their own geometry:
+
+                            tileset
```

Comment:
Although I'm a big ascii art fan for docs, maybe we could try some UML [class/instance diagrams](https://www.uml-diagrams.org/class-diagrams-overview.html) for explaining the LOA/LOD/TILE hierarchy ?
We could make some trial with e.g. [PlantUML](https://github.com/VCityTeam/UD-SV/blob/master/Tools/ToolForDiagramming.md#plantuml) (supported by Gitlab/Github)...